HACKING.md

```diff
@@ -269,6 +269,25 @@ Attaching to current-bench_db_1, current-bench_db-migrate_1, current-bench_pipel
 ...
 ```
 
+## Database schema
+
+Currently, we have two tables. We want to eventually phase out to have 
+two separate tables, one that stores the metadata and one that stores the
+metrics.
+
+The metadata table contains:
```

Comment:
I'm not sure if we really want to have a description of the DB schema in the docs. It can easily be out of sync with the actual state of the DB.